initrd/bin/flash.sh

```diff
@@ -77,17 +68,43 @@ flashrom_progress() {
         if [ "$status" == "reading" ]; then
             if echo "${IN}" | grep "done." > /dev/null ; then
                 status="writing"
+                IN=
             fi
         fi
         if [ "$status" == "writing" ]; then
-            echo -ne "Flashing: [${progressbar}${spin:$spin_idx:1}${progressbar2}] (${percent}%)\\r"
-            if echo "$IN" | grep "Verifying" > /dev/null ; then
+            #if [ "$prev_prev_word" = "writing" ] && [ "$prev_word" = "range" ]; then
+            #    # "... is writable, writing range (0x000000..0x000fff)"
+            #    current=$(echo "$IN" | sed -nE 's/^\(0x[0-9a-f]+..(0x[0-9a-f]+)\)\.$/\1/p')
+            #    if [ "$current" != "" ]; then
+            #        percent=$((100 * (current + 1) / total_bytes))
+            #        echo "current: $current, total: $total_bytes, percent: $percent"
+            #        print_flashing_progress "$percent"
+            #    else
+            #        echo "failed to match: $IN"
+            #    fi
+            #fi
```

Comment:
@tlaurion Could you delete this commented-out code?  Sorry this came from my WIP branch.  (Keep the walk_eraseblocks() notes below though please.)

Reply:
Done under #1423, will rebase when CircleCI actually builds a cache that it can reuse.